lib/src/utils.dart

```diff
@@ -35,3 +35,28 @@ final String _sdkDir = (() {
   assert(FileSystemEntity.isFileSync(p.join(aboveExecutable, 'version')));
   return aboveExecutable;
 })();
+
+bool isFlutterSdk() {
```

Comment:
could/should this be a final field, too?

Reply:
👍